config/webpack.base.config.js

```diff
@@ -0,0 +1,44 @@
+const webpack = require("webpack");
+const merge = require("webpack-merge");
+
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = env => {
+  const { PLATFORM, VERSION } = env;
+  return merge([
+      {
+        module: {
+          rules: [
+            {
+              test: /\.js$/,
+              exclude: /node_modules/,
+              use: {
+                loader: 'babel-loader'
+              }
+            },
+            {
+              test: /\.scss$/,
```

Comment:
Add rules for images to import them like 
```
import Logo from “./logo.png”;
```

Reply:
Done. Add image loader.